LibTrixi.jl/src/api_c.jl

```diff
@@ -328,32 +373,60 @@ trixi_nvariables_cfptr() = @cfunction(trixi_nvariables, Cint, (Cint,))
 
 
 """
-    trixi_load_cell_averages(data::Ptr{Cdouble}, simstate_handle::Cint)::Cvoid
+    trixi_load_cell_averages(data::Ptr{Cdouble}, index::Cint, simstate_handle::Cint)::Cvoid
```

Comment:
```suggestion
    trixi_load_cell_averages(data::Ptr{Cdouble}, variable_id::Cint, simstate_handle::Cint)::Cvoid
```
`index` is super generic, I suggest to use something more specific (I think `variable_id` or similar is what we use in Trixi.jl). Also, it might be good to add a comment to the docstring on how to obtain the minimum and maximum variable_id, and to communicate the fact that this is one-based indexing.